.gitlab/package_build/deb.yml

```diff
@@ -148,10 +144,7 @@ agent_deb-arm64-a7:
     - echo "Target architecture ${DD_TARGET_ARCH:=amd64}"
     # remove artifacts from previous pipelines that may come from the cache
     - rm -rf $OMNIBUS_PACKAGE_DIR/*
-    # Artifacts and cache must live within project directory but we run omnibus in a neutral directory.
-    # Thus, we move the artifacts at the end in a gitlab-friendly dir.
     - !reference [.setup_deb_signing_key]
-    # Use --skip-deps since the deps are installed by `before_script`.
```

Comment:
If this is no longer true, do we know why we still use `--skip-deps`?

Reply:
This one is still true, but:
- I think it's easy to read why it is used (the `- !reference [.retrieve_linux_go_deps]` part is a few lines above)
- I find it weird that there is only a comment for that specific option, and not all the other ones that fall in the same pattern (eg. `--go-mod-cache`).

So I proposed removing it (assuming that reading the job script is enough to understand), but another option is to put it back, and add the explanations for the other options as well, to be consistent.

Reply:
👍 Removing it is fine.